tests/gleam.test.zsh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env zsh
+
+# Required for shunit2 to run correctly
+setopt shwordsplit
+SHUNIT_PARENT=$0
+
+# ------------------------------------------------------------------------------
+# SHUNIT2 HOOKS
+# ------------------------------------------------------------------------------
+
+oneTimeSetUp() {
+  export TERM="xterm-256color"
+  export PATH=$PWD/tests/stubs:$PATH
+
+  GLEAM_VERSION="1.5.1"
```

Comment:
`gleam` binary probably needs to be mocked. Please refer to other binary mocks as an example.